scripts/deploy.js

```diff
@@ -748,6 +760,7 @@ async function buildDeploymentActionPlan(plan, config) {
     //   It may require some changes, or precomputing addresses.
     await ensureVerifierDeployment(plan, config);
     await ensureInitialRoot(plan, config);
+    await ensureStateBridgeDeployment(plan, config);
```

Comment:
Given the rename mentioned above you can move this up with the `get*` functions above. I'd put it under `getWallet`.

Reply:
fixed